_faq/13-patch-wfly-with-weld.asciidoc

```diff
@@ -0,0 +1,26 @@
+= How to patch Wildfly with newer Weld version?
+
+Each version of Wildfly is shipped with given version of Weld. However, with every Weld release we also release a patch for Wildfly which allows you to easily upgrade Weld version.
```

Comment:
New line before "However".
"However, for every Weld release we also prepare a patch for WildFly..." ?